plugins/osdn/vnids.go

```diff
@@ -34,7 +69,7 @@ func (oc *OsdnController) GetVNID(name string) (uint, error) {
 		return id, nil
 	}
 	// In case of error, return some value which is not a valid VNID
-	return MaxVNID + 1, fmt.Errorf("Failed to find netid for namespace: %s in vnid map", name)
+	return netid.MaxVNID + 1, fmt.Errorf("Failed to find netid for namespace: %s in vnid map", name)
```

Comment:
Yes, returned uint shouldn't matter in case of error but to be on safer side, I'm trying to avoid the case where by mistake someone ignores the error and tries to use this value. 0 is valid vnid(admin network ID) and MaxVNID is  2^24, so we don't expect overflow and we have vnid range checks before persisting/assigning these ids.